src/components/Sidebar.native.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import React from 'react'
+import { useSelector } from 'react-redux'
+import _ from 'lodash'
+import { findTreeDescendants } from '../util/recentlyEditedTree'
+import { State } from '../util/initialState'
+// import RecentlyEditedBreadcrumbs from './RecentlyEditedBreadcrumbs'
+import { View, Text, StyleSheet } from 'react-native'
+
+/** Displays recently edited thoughts with a header. */
+const RecentEdited = () => {
+  const recentlyEditedTree = useSelector((state: State) => state.recentlyEdited)
+  const showHiddenThoughts = useSelector((state: State) => state.showHiddenThoughts)
+  // eslint-disable-next-line fp/no-mutating-methods
+  const recentlyEdited = _.reverse(
+    _.sortBy(findTreeDescendants(recentlyEditedTree, { startingPath: [], showHiddenThoughts }), 'lastUpdated'),
+  )
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.headerText}>Recently Edited Thoughts</Text>
+
+      <View style={styles.padding}>
+        <Text style={styles.text}>RecentlyEditedBreadcrumbs will go here</Text>
+        {/* {recentlyEdited.map((recentlyEditedThought, i) => (
```

Comment:
is this still pending conversion?

Reply:
yip

Reply:
will do it separated the `RecentlyEditedBreadcrumbs` component has some other links that require a bit of thinking so I can refactor it correctly.